apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -80,9 +87,9 @@ const DebugConsole = connect(
         this.props.removeWatchExpression(
           input.substring(UNWATCH_COMMAND_PREFIX.length)
         );
-      } else if (this.props.jsInterpreter) {
+      } else if (this.props.isAttached) {
```

Comment:
Nice improvement! Seems better than passing in something that could be all 4 states